test/CrateTest.js

```diff
@@ -786,4 +789,46 @@ describe('Crate', function() {
       })
     })
   })
+
+  context('where the file is stored', function(){
```

Comment:
this doesn't follow your format. Would you prefer I update the tests to match your format?

I did it my way so I could get it working